src/core/sliding-window/index.ts

```diff
@@ -105,7 +105,8 @@ export async function truncateConversationIfNeeded({
 	let error: string | undefined
 	let cost = 0
 	// Calculate the maximum tokens reserved for response
-	const reservedTokens = maxTokens || contextWindow * 0.2
```

Comment:
I think we need to update this to use the shared logic instead. I thought I got all the spots but looks like I missed this one.